### ISSUE TITLE
Bucket Policy Permissions

### ISSUE THREAD

**Issue by ju2wheels** [sonatype/nexus-blobstore-s3#44]
Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**
Create initial blobstore on an existing empty S3 bucket

* **What feature or behavior is this required for?**
S3 blobstore

* **How could we solve this issue? (Not knowing is okay!)**
The recommended S3 bucket policy doesnt seem to cover what the plugin actually needs permission wise.

I have the policy applied but it also seems to want 'PutBucketLifecycle' as well. Can you please provide an updated list of required policy permissions required?

* **Anything else?**

**Comment by ataylor284** [comment]
According to https://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html, putting the bucket lifecycle configuration is covered by `s3:PutLifecycleConfiguration` which is already listed.  What error are you seeing?

**Comment by ju2wheels** [comment]
It couldnt successfully use the bucket otherwise and CloudTrail reported the error as PutBucketLifecycle denied when I checked.

I just tried with the minimal config in the README, exactly as it is there and it seems to work fine. I originally had the same defined across two rules one for read perms and one for write perms so there may have been a problem in how I defined it there. Seems ok though, closing.
